### ISSUE TITLE
搜索功能貌似无效

### ISSUE THREAD

**Issue by wwjj999** [n4ze3m/page-assist#9]
搜索功能貌似无效，搜索后等待几十分钟都无反映

**Comment by n4ze3m** [comment]
Hey, which embedding model are you using? Search may slow down if the embedding model you used is the same as the chat model. I highly recommend using Nomic Embed Text (https://ollama.com/library/nomic-embed-text) as the embedding model, which will be faster.

**Comment by n4ze3m** [comment]
Also, Internet search is in beta and will improve in upcoming updates.

**Comment by Lemonlee8** [comment]
I set my embedding model as Nomic Embed Text, and internet search function works well. But, as you had said, it's a little bit slow and inaccurate.

**Comment by n4ze3m** [comment]
Yes, it's slow. I'm adding a new feature where answers are sourced directly from Google search queries instead of RAG. This feature can be enabled or disabled on the settings page.

I think the inaccuracy issue may be due to my simple vector search. I will explore another solution. Thank you for reporting it.

**Comment by wwjj999** [comment]
Thank you very much for your updateWe will continue to try and hope that your product will become more and more perfect.
Additionally, we would like to add a hold button to all optional parameters in the product settings options.



---- 回复的原邮件 ----
| 发件人 | Muhammed ***@***.***> |
| 日期 | 2024年03月01日 16:04 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [n4ze3m/page-assist] 搜索功能貌似无效 (Issue #9) |

Yes, it's slow. I'm adding a new feature where answers are sourced directly from Google search queries instead of RAG. This feature can be enabled or disabled on the settings page.

I think the inaccuracy issue may be due to my simple vector search. I will explore another solution. Thank you for reporting it.

**Comment by wwjj999** [comment]
https://github.com/sugarforever/chat-ollama
The content of this code repository is good,I suggest you.



| |
***@***.***
|
|
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年03月01日 14:19 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [n4ze3m/page-assist] 搜索功能貌似无效 (Issue #9) |

I set my embedding model as Nomic Embed Text, and internet search function works well. But, as you had said, it's a little bit slow and inaccurate.
